pkg/sinit/init.go

```diff
@@ -137,6 +137,9 @@ var handlerMulti *strc.MultiHandler
 var handlerSplunk *splunk.SplunkHandler
 var handlerCloudWatch *cloudwatchwriter2.Handler
 
+// The main logger instance. Use InitializeLogging to create it.
+var logger *slog.Logger
```

Comment:
do we need to have global logger instance?
Sounds like it's just for pgx, could we avoid it ?